### ISSUE TITLE
Crashing to desktop at startup - Trails FC GOG version

### ISSUE THREAD

**Issue by DooMGuy87** [ZhenjianYang/SoraVoice#2]
As the title says, the game crashes at boot after installing Sora Voice (lite). I have followed the instructions and put the ogg (with the audio files) and the scena folders inside the voice folder on the game directory but the game crashes at startup. it's strange because the old soravoice mod worked perfectly before.
Tried with the dinput8 dll and the dsound dll (not at the same time) and had the same result. After removing those dlls, the game starts normally.
My game version is 2019.12.05
Thanks in advance.

**Comment by XanaduCSIV** [comment]
Yes, can confirm this exact behavior.  And this isn't limited to the GOG version of the games either.  Happens on a Steam install also.

Also, this happens in FC and TC.  SC manages to work just fine with both dinput8.dll and dsound.dll.  I'll log another issue for TC to keep track of it.

**Comment by MartinF99** [comment]
happens for me now too, Trails in the sky FC crashes on the first cutscene

**Comment by MartinF99** [comment]
Found out something else, the game does not crash when launched directly from The exe, but always crashes when launched through GOG Galaxy (The launcher)

**Comment by a-rabin** [comment]
I have this issue even directly running the EXE. And I've used Skyinstaller.

**Comment by Sargon-Aelther** [comment]
For people with the crashing issue: Try using "dsound.dll" instead of "dinput8.dll" (Don't use both together). This will allow you to use Galaxy for overlay and achievements.

For the dev; Please explain the differences between "dsound.dll" and "dinput8.dll", because it seems to me that "dsound.dll" is better.
